src/com/reco1l/osu/data/Scores.kt

```diff
@@ -0,0 +1,197 @@
+@file:JvmName("Scores")
+package com.reco1l.osu.data
+
+import androidx.room.Dao
+import androidx.room.Entity
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.PrimaryKey
+import androidx.room.Query
+import org.json.JSONObject
+import ru.nsu.ccfit.zuev.osu.scoring.StatisticV2
+
+
+@Entity
+data class ScoreInfo @JvmOverloads constructor(
+
+    /**
+     * The score ID.
+     */
+    @PrimaryKey(autoGenerate = true)
+    val id: Long = 0,
+
+    /**
+     * The beatmap file name.
+     */
+    val beatmapFilename: String,
```

Comment:
As far as I know, `filename` in the old score table represents the beatmap's path. Maybe this column would better be named `beatmapPath`?

Reply:
Resolved with [742592e](https://github.com/osudroid/osu-droid/pull/390/commits/742592e67008b14badf7568da20c9c72a4896a8c)